src/components/DimensionMenu.js

```diff
@@ -12,7 +12,7 @@ import { getAvailableAxes } from '../modules/layoutUiRules'
 import { AXIS_ID_COLUMNS, AXIS_ID_FILTERS } from '../modules/layout/axis'
 import { DIMENSION_ID_DATA } from '../modules/predefinedDimensions'
 import { isDualAxisType, getDisplayNameByVisType } from '../modules/visTypes'
-import { getAxisName } from '../modules/axis'
+import { getAxisNameByVisType } from '../modules/axis'
 
 const getAxisItemLabelPrefix = isDimensionInLayout =>
     isDimensionInLayout ? 'Move to' : 'Add to'
```

Comment:
These will (1) never be translated and (2) lack the context required to translate them correctly.  Since these are only used 2 places, this function should instead look like this:

```js
const getAxisItemLabel = (axisName, isDimensionInLayout) => 
    isDimensionInLayout 
      ? i18n.t('Move to {axisName}', { axisName })
      : i18n.t('Add to {axisName}', { axisName })
```

And the two references to this function below should be updated.

Reply:
Indeed, same issue as found in https://github.com/dhis2/data-visualizer-app/pull/623, fixed.